### ISSUE TITLE
Missing files?

### ISSUE THREAD

**Issue by YubinXie** [theislab/scArches-reproducibility#1]
Hi,

I am trying to check the script and I did not find DataDownloader.py and ModelTrainer.py in the folder. It would be great if you can have a look.

```
cd scripts/
python DataDownloader.py
python ModelTrainer.py all
```

**Comment by M0hammadL** [comment]
Hi @YubinXie, sorry for the inconvenience, some of the links are not working, we are re-uploading some of them. we will fix it today or tomorrow. We will let you know here

**Comment by Naghipourfar** [comment]
Hi @YubinXie, 

We have re-uploaded all necessary datasets and latent space representations files needed to reproduce the results and run the notebooks. For the method comparison figures, We have uploaded Pancreas and Mouse Brain latent space representations of trVAE, scArches trVAE, Conos, Harmony, scVI, Liger, MNNCorrect, Scanorama, and Seurat V3 in `latent space comparison` subdirectory inside `Benchmark Scores`. Also, KNN and EBM scores of each method for Pancreas and Mouse Brain datasets are uploaded in `Benchmark Scores` subdirectory inside `Benchmark results` directory. These files can be used to run the method-comparison related notebooks (figure 2, supplementary figure 4-6). 
Also Integrated latent space representation of Tabula Muris Senis, PBMC Covid19, and HCL_MCA related to figures 3, and 4 (and supplementary figures 7-10) have been uploaded. They are necessary to run the related notebooks in which UMAPs, KNN label transfer, and some downstream analysis are performed. Also, we have updated notebooks and scripts to be compatible with the new release of scArches API.

All datasets (h5ad files) can be downloaded from [here](https://drive.google.com/drive/folders/1QQXDuUjKG8CTnwWW_u83MDtdrBXr8Kpq?usp=sharing).
All latent space h5ad files can be downloaded from [here](https://drive.google.com/drive/folders/1QqNCBv19-iF9whYIMkVL6DyBRH0x_G57?usp=sharing).
